fallout/jobs/job_types/legion/centurion.dm

```diff
@@ -0,0 +1,29 @@
+/datum/job/f13/legion/centurion
+	title = "Centurion"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "Praetorians, The Legate, and Caesar."
+	description = "Command your Centura, leading them to glorious victory at all costs."
+	enforce = "The Legion expects you to lead them effectively. Do not be a coward."
+
+	outfit = /datum/outfit/job/f13/legion/centurion
+
+	display_order = JOB_DISPLAY_ORDER_F13_CENTURION
+
+/datum/outfit/job/f13/legion/centurion
+	name = "_Legion Centurion"
+	jobtype = /datum/job/f13/legion/centurion
+
+	head = /obj/item/clothing/head/helmet/fallout/legion/centurion
+	glasses = /obj/item/clothing/glasses/sunglasses/big
+	backpack = /obj/item/storage/backpack/satchel/leather
+	uniform = /obj/item/clothing/under/fallout/legion/fatigues/redpadded
+	shoes = /obj/item/clothing/shoes/fallout/military/legion/darkboots
+	suit = /obj/item/clothing/suit/armor/fallout/legion/centurion
+	suit_store = /obj/item/gun/ballistic/automatic/fallout/smg10mm
+	r_hand = /obj/item/melee/powerfist
```

Comment:
Ideally I'd like for Cent to not have a powerfist any longer, but this really depends on whether we end up having Speculatore be an admin spawn, Prae be an admin spawn, or neither be an admin spawn. 

Basically, if Speculatores end up being the normal role, we should probably tone down the ranged capabilities of Cent. If Praes end up being the normal role, we should probably tone down the melee capabilities of Cent. If both, I'm going to attack someone.